doc/zmq.md

```diff
@@ -102,7 +104,20 @@ is structured as the following based on the type of message:
 
 Where the 8-byte uints correspond to the mempool sequence number.
 
-These options can also be provided in bitcoin.conf.
+`rawtx`: notifies about all transactions, whether when they are added to mempool or when a new block arrives. This means it could happen that a transaction is published multiple times. First when it enters the mempool and then again in each block that includes it. The body of the message is the serialized transaction.
```

Comment:
nit, `-zmqpubrawtx` instead of `rawtx`? (same thing below)

Reply:
I am not sure. I think that `rawtx` is the ZMQ topic that the subscriber receives notification about and `-zmqpubrawtx` is the argument used to set the network address of this topic in node daemon (`bitcoind`).

If that is not the case, I can change them to `-zmq*`.

Reply:
I'd say: leave it as `rawtx` (the topic). This sections starts in line 96 with `The topics are:`.

Reply:
We should mention that this is a multipart message too. 

```suggestion
`rawtx`: Notifies about all transactions, both when they are added to mempool or when a new block arrives. This means a transaction could be published multiple times. First, when it enters the mempool and then again in each block that includes it.The messages are ZMQ multipart messages with three parts. The first part is the topic (`rawtx`), the second part is the serialized transaction, and the last part is a sequence number (to detect lost messages).

| rawtx | <serialized transaction> | <uint32 sequence number in Little Endian>
```